src/compiler/types.ts

```diff
@@ -5241,6 +5241,11 @@ namespace ts {
         writeFile: WriteFileCallback;
     }
 
+    export const enum LexicalEnvironmentScoping {

```

Comment:
I think `LexicalEnvironmentKind` would be more consistent with TypeScript's internal naming conventions.